utils/src/eda_helper_utils/utils_analysis.py

```
@@ -0,0 +1,154 @@
+def harmonize_metadata(
+    df: pd.DataFrame, modality: str, patient_dir_name: str
+) -> pd.DataFrame:
+    """Rename 2D/3D metadata columns to a common schema and add patient/timepoint/modality columns.
+
+    Common schema columns added: Metadata_patient, Metadata_timepoint,
+    Metadata_patient_tumor (original directory name), Metadata_modality.
+    """
+    df = df.copy()
+    rename_map = HARMONIZE_2D_MAP if modality == "2D" else {}
+    df = df.rename(columns={k: v for k, v in rename_map.items() if k in df.columns})
+    patient, timepoint = patient_timepoint(patient_dir_name)
```

Ruff (0.16.3)

[warning] Unpacked variable `patient` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)